packages/protocol/contracts/rewards/StakingRewards.sol

```diff
@@ -574,18 +668,23 @@ contract StakingRewards is ERC721PresetMinterPauserAutoIdUpgradeSafe, Reentrancy
 
     require(block.timestamp >= position.lockedUntil, "staked funds are locked");
 
+    uint256 effectiveAmount = toEffectiveAmount(
+      amount,
+      position.baseTokenExchangeRate,
+      getEffectiveAmountMultiplier(position.positionType)
```

Comment:
Hmm so I'm realizing a problem with not storing the multiplier. Situation is when multiplier is changed via governance while positions are already staked. If this happens, then the effective supply would have been adjusted up by the old multiplier when the positions were staked, but then adjusted down by the new multiplier on unstake, resulting in incorrect total staked supply and reward amount. 

So I'm not sure this is going to work... the best alternative I can think of is to store the multiplier after all. If we want to support existing positions getting the new multiplier, we could provide a function that lets you adjust your multiplier? Like, looks at your position's stored multiplier and the global one, and properly adjusts staked supply for you? If we decided to adjust the multiplier down, there would be no incentive / easy way to force old position's multipliers down, unless we called manually. The situation is similar to `kick`.

@blakewest @emilyhsia thoughts?